juriscraper/pacer/appellate_docket.py

```diff
@@ -362,7 +334,7 @@ def parties(self):
             #  <B>Terminated: </B>07/31/2017<BR>
             #  Respondent
 
-            name_role = self.redelimit_p(cells[0], r'(?i)<br\s*/?>')
+            name_role = self.redelimit_p(cells[0], self.BR_REGEX)
```

Comment:
I guess the other way to do this is to make it a default arg for the method. Seems tidier to me, but not a strong feeling.

Reply:
I don't think so. The method is abstract. The use of BR is only one particular thing. If we were to make that the default it would really be a different kind of method and should have a different name and maybe be in a different place and all that kind of stuff.